static/css/main.css

```diff
@@ -1,17 +1,20 @@
-*{
-    padding: 0;
-    margin: 0;
-    box-sizing: border-box;
-    scroll-behavior: smooth;
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@300;500&family=Kalam:wght@300;400;700&family=Poppins:wght@100;200;300;400;500&family=Roboto:wght@100;300;400;500&family=Rubik:wght@300;400;500;700&display=swap");
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+  font-family: Poppins;
 }
 
-:root{
-    --primary-color: #FFFFFF;
-    --accent-color: #4A2DFC;
-    --secondary-color: #0C0101;
-    --light-grey: #D9D9D9;
-    --mid-grey: #7A7A7A;
-    --dark-grey:  #353535;
+:root {
+  --accent-color: #4a2dfc;
+  --primary-color: #ffffff;
+  --secondary-color: #0c0101;
+  --light-grey: #d9d9d9;
+  --dark-grey: #353535;
+  --lightgrey: #f0f0f0;
+  --mid-grey: #7a7a7a;
```

Comment:
Use previous changes:
    --primary-color: #FFFFFF;
    --accent-color: #4A2DFC;
    --secondary-color: #0C0101;
    --light-grey: #D9D9D9;
    --mid-grey: #7A7A7A;
    --dark-grey:  #353535;

from line 11 to 16 use the above code(previous changes)